velox/connectors/hive/storage_adapters/s3fs/tests/S3FileSystemTest.cpp

```diff
@@ -340,4 +340,31 @@ TEST_F(S3FileSystemTest, registerCredentialProviderFactories) {
           }),
       "CredentialsProviderFactory 'my-credentials-provider' already registered");
 }
+
+TEST_F(S3FileSystemTest, exist) {
```

Comment:
Let's test this along with the existing test `writeFileAndRead`.

Reply:
@majetideepak Yes. Updated.